onthequad/onthequad/UserEndPoints.py

```diff
@@ -28,37 +28,39 @@ def on_post(self, req, resp):
             try:
 
                 email = data['email']
+                #Create userName
```

Comment:
user authentication is a method from Firebase. We just call it in and use it. I think creating users data and users authentication should go together so it is easier to keep track of.

Reply:
I think to build upon Justin's comment, it might be beneficial to split those method calls into separate methods. This is good for a couple reasons:

Testing - Makes it way easier to test small, granular methods. 
SRP- (Single Responsibility Principle)

Reply:
Did that